engines/groovie/detection.cpp

```diff
@@ -180,7 +191,21 @@ static const GroovieGameDescription gameDescriptions[] = {
 			Common::EN_ANY, Common::kPlatformMacintosh, ADGF_UNSTABLE,
 			GUIO4(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT)
 		},
-		kGroovieV2, 1
+		kGroovieT11H, 1
+	},
+
+	// The 11th Hour Windows English (Available on Steam)
```

Comment:
Adding this Digital distribution is a duplicate of the Mac/Windows detection entry higher up, so ScummVM asks me which it is.